hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -503,8 +503,8 @@ message BucketInfo {
     optional string sourceVolume = 12;
     optional string sourceBucket = 13;
     optional uint64 usedBytes = 14;
-    optional uint64 quotaInBytes = 15;
-    optional uint64 quotaInNamespace = 16;
+    optional int64 quotaInBytes = 15 [default = -2];
```

Comment:
https://github.com/apache/ozone/pull/1677#issuecomment-745019611
Any reason for moving back to int64?

Reply:
Using sint64, I found that the default value changed to -9223372036854775808 after switching to long. This could be a compatibility issue. So I switched to int64.
![image](https://user-images.githubusercontent.com/13825159/102593001-4291b800-414f-11eb-84b8-3bd0b33290a6.png)

Reply:
Interesting, thanks for info.